django_stomp/helpers.py

```diff
@@ -52,3 +53,35 @@ def only_destination_name(destination: str) -> str:
 
 def create_dlq_destination_from_another_destination(destination: str) -> str:
     return f"DLQ.{only_destination_name(destination)}"
+
+
+def remove_key_from_dict(dictionary, key):
+    dictionary.pop(key, None)
+
+
+def is_destination_from_virtual_topic(destination_name: str) -> bool:
+    return ".VirtualTopic." in destination_name
+
+
+def is_dlq_destination(destination_name: str) -> bool:
+    return "DLQ." in destination_name
```

Comment:
This prefix can be changed, it would be interesting to say that it is important not to change it and in the future make it configurable